CMakeLists.txt

```diff
@@ -14,6 +14,8 @@
 cmake_minimum_required(VERSION 3.28)
 message(STATUS "Building using CMake version: ${CMAKE_VERSION}")
 
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
Where do we need this?

Reply:
Agreed, it is unset by default, if it is set by one of the dependencies please manage that in the relevant cmake file.

Reply:
I met below error in the workflow in `Make Debug Build` of `Ubuntu debug with resolve_dependency`, but am not clear why it only occurs in this PR. It would be appreciated if you have any clue. Thanks.

```
  "velox_exception" of type SHARED_LIBRARY
    depends on "velox_process" (weak)
    depends on "velox_file" (weak)
  "velox_common_base" of type SHARED_LIBRARY
    depends on "velox_exception" (weak)
    depends on "velox_process" (weak)
    depends on "velox_file" (weak)
    depends on "velox_common_compression" (weak)
    depends on "velox_test_util" (weak)
  "velox_common_compression" of type SHARED_LIBRARY
    depends on "velox_exception" (weak)
    depends on "velox_process" (weak)
    depends on "velox_file" (weak)
  "velox_file" of type SHARED_LIBRARY
    depends on "velox_exception" (weak)
    depends on "velox_process" (weak)
    depends on "velox_common_base" (weak)
  "velox_process" of type SHARED_LIBRARY
    depends on "velox_file" (weak)
    depends on "velox_exception" (weak)
  "velox_test_util" of type SHARED_LIBRARY
    depends on "velox_exception" (weak)
    depends on "velox_process" (weak)
    depends on "velox_file" (weak)
```

Reply:
This change was removed because the workflow now works without it.